extensions/2.0/Vendor/EXT_mesh_features/schema/class.property.schema.json

```diff
@@ -58,22 +111,38 @@
     },
     "normalized": {
       "type": "boolean",
-      "description": "Specifies whether integer values are normalized. This applies both when `componentType` is an integer type. For unsigned integer component types, values are normalized between `[0.0, 1.0]`. For signed integer component types, values are normalized between `[-1.0, 1.0]`. For all other component types, this property must be false.",
+      "description": "Specifies whether integer values are normalized. This applies when `componentType` is an integer type. For unsigned integer component types, values are normalized between `[0.0, 1.0]`. For signed integer component types, values are normalized between `[-1.0, 1.0]`. For all other component types, this property must be false.",
       "default": false
     },
     "max": {
       "oneOf": [
-        {"type": "number"},
-        {"type": "array", "items": {"type": "number"}, "minItems": 1}
+        {
+          "type": "number"
+        },
+        {
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 1
+        }
       ],
-      "description": "Maximum allowed value for the property. Only applicable for single-value numeric types, fixed-length arrays of numeric types, and `VECN` types. For single-value numeric types this is a single number. For fixed-length arrays and `VECN`, the maximum is an array with the same number of elements. The `normalized` property has no effect on the maximum, which always contains integer values."
+      "description": "Maximum allowed value for the property. Only applicable for single-value numeric types, fixed-length arrays of numeric types, `VECN`, and `MATN` types. For single-value numeric types this is a single number. For fixed-length arrays, `VECN`, and `MATN` types, the maximum is an array with the same number of elements. The `normalized` property has no effect on the maximum, which always contains integer values."
```

Comment:
The 'minimum' and 'maximum' of a matrix are not obviously well-defined (minimum rotation? scale? translation?) so perhaps it's worth mentioning that for array, vector, and matrix types this is a component-wise min/max?